### DISCUSSION TITLE
`Replit-v2-CodeInstruct` vs `Replit-v1-CodeInstruct`

### DISCUSSION THREAD

**Discussion by matthiasgeihs** [abacaj/code-eval#5] [Q&A]
Why does `Replit-v2-CodeInstruct` perform significantly worse than `Replit-v1-CodeInstruct`?
(21.5% vs 25.8% pass@1)

**Comment by abacaj** [comment]
Hi, v1 model is trained on 512 sequence length. V2 is trained on 2000 sequence length. The longer sequence training from what I know wasn’t as effective because of the training data used

**Reply by matthiasgeihs** [reply]
Yes, I've seen this, but it shouldn't make the results worse?
Maybe @teknium1 can weigh in here?
